sdk/core/azure-core/azure/core/utils/_pipeline_transport_rest_shared.py

```diff
@@ -409,3 +410,9 @@ def _aiohttp_body_helper(
         response._decompressed_content = True
         return response._content
     return response._content
+
+
+def get_file_items(files: "FilesType") -> Sequence[Tuple[str, "FileType"]]:
```

Comment:
```suggestion
def get_file_items(files: "FilesType") -> Iterable[Tuple[str, "FileType"]]:
```

Iterable might work here instead of Sequence due to the `ItemsView`, though typically you want to type returns as exact.

Reply:
yeah, the thing is it's more of a `Sequence` vs an `Iterable`, bc there is an explicit order of things. Honestly I'm not too sure why `ItemsView` isn't a `Sequence`, bc for the Python versions we're using, it does have order. I went ahead and casted it, lmk if you have any objections

Reply:
I'm fine with the cast for now, we can just add a comment on why its being done and perhaps fixme later (we can always break typing)

perhaps an issue in the mypy repo, because ItemView should be a sequence